README.md

```diff
@@ -1,6 +1,6 @@
 # Commodore Component: ingress-nginx
 
-This is a [Commodore][commodore] Component for ingress-nginx.
+This is a [Commodore][commodore] Component for the NGINX Ingress Controller.
```

Comment:
Might be worth to express which one. There is the one of the Kubernetes community and the commersion one from NGINX.

Reply:
Updated that we are using there the variant from the Kubernetes Community.